client/asset/dcr/dcr.go

```diff
@@ -734,12 +742,17 @@ func (dcr *ExchangeWallet) split(value uint64, coins asset.Coins, inputsSize uin
 	// `-4: rejected transaction: transaction in the pool already spends the
 	// same coins` error.
 	// // Unlock the spent coins.
-	// err = dcr.ReturnCoins(coins)
+	// err = dcr.returnCoins(coins)
 	// if err != nil {
 	// 	dcr.log.Errorf("error locking coins spent in split transaction %v", coins)
 	// }
 
-	dcr.logSplitFunds(op, fundingCoins)
+	// Associate the funding coins with the split tx output, so that the coins
+	// can be unlocked when the swap is sent. The helps to deal with a timing
+	// issue with dcrwallet where listunspent might still return outputs that
+	// were just spent in a transaction broadcast with sendrawtransaction.
+	// Instead, we'll keep them locked until the split output is spent.
+	dcr.splitFunds[op.pt] = fundingCoins
```

Comment:
Was the only call site for the now-removed `logSplitFunds`, and we want several of the above operations to be atomic from the perspective of all coin maps and wallet utxo views.